lib/rules/member-delimiter-style.js

```diff
@@ -13,11 +14,22 @@ const definition = {
     properties: {
         delimiter: { enum: ["none", "semi", "comma"] },
         requireLast: { type: "boolean" },
-        ignoreSingleLine: { type: "boolean" },
+        singleLine: { enum: ["none", "semi", "comma"] },
     },
     additionalProperties: false,
 };
```

Comment:
i was thinking about this rule, and i feel like this will be more consistent with how eslint specifies options.
```
{
  "multiline": {
    delimiter: { enum: ["none", "semi", "comma"] },
    requireLast: { type: "boolean" },
  },
  "singleline": {
    delimiter: { enum: ["none", "semi", "comma"] },
    requireLast: { type: "boolean" },
  }
}
```

Reply:
```ts
const foo: { a: number } = { a: 1 }
```

```ts
interface foo {
   a: number,
}
```

Reply:
I think that's a good idea.
I wanted to do a more drastic change to the config in a separate PR (this one already has ~700 changed lines).

Because this is breaking, I am blocking the 1.0.0 release till the second PR is done.